packages/dx-react-grid/src/plugins/column-chooser.tsx

```diff
@@ -96,21 +101,15 @@ export class ColumnChooser extends React.PureComponent {
   }
 }
 
-ColumnChooser.propTypes = {
-  overlayComponent: PropTypes.func.isRequired,
-  containerComponent: PropTypes.func.isRequired,
-  itemComponent: PropTypes.func.isRequired,
-  toggleButtonComponent: PropTypes.func.isRequired,
-  messages: PropTypes.object,
-};
-
-ColumnChooser.defaultProps = {
+ColumnChooserBase.defaultProps = {
```

Comment:
This one and the following `components` can be initialized right where declared.